bike_rent/views/sale_views.xml

```diff
@@ -0,0 +1,11 @@
+<odoo>
+
+<record id="bike_rent_sale_action" model="ir.actions.act_window">
+    <field name="name">Quotation</field>
+    <field name="res_model">sale.order</field>
+    <field name="view_mode">tree,form</field>
+</record>
+
+<menuitem name="Quotation" id="bike_rent.menu_1_sale" parent="bike_rent.menu_1" action="bike_rent_sale_action"/>
```

Comment:
1. here too,
2. inconsistency in references, either declare full references, or don't, but stay consistent.